druid/base/scroll.lua

```diff
@@ -180,7 +192,8 @@ function Scroll.init(self, view_node, content_node)
 	self.drag.on_touch_end:subscribe(self._on_touch_end)
 
 	self.hover = self.druid:new_hover(view_node)
-	self.hover.on_mouse_hover:subscribe(self._on_mouse_hover)
+	self.hover.on_mouse_hover:subscribe(on_scroll_mouse_hover)
```

Comment:
The on_mouse hover calls even it's hover or going away already
The on_mouse_away function is redundant